src/ChannelCallback.h

```diff
@@ -0,0 +1,12 @@
+#include "Bindings/PluginLua.h"
+
+class cChannelCallback
```

Comment:
Man, callbacks? In C++11, llamas!

uh

lambdas are the new rad!

----

Is this possible with lambdas, so you don't introduce an extra class + two files? Though again, I remember `cLuaState::cCallback` being xoft's creation.

If so, xoft, can't lua directly pass functions to C++?